### PR TITLE
Add a initialization toggle to the Helm chart #3771

### PR DESCRIPTION
**Pull request by wromansky**

### CODE REVIEW

**Comment by nicknezis** [comment]
This fixes #3771

**Review by nicknezis — CHANGES_REQUESTED** [review]

File: deploy/kubernetes/helm/values.yaml.template [hunk 1]
```diff
@@ -33,6 +33,8 @@ image: apache/heron:VERSION
 # Heron image pull policy
 imagePullPolicy: IfNotPresent
 
+initialize: false
```
Comment (nicknezis) [inline deploy/kubernetes/helm/values.yaml.template:36]: You forgot to remove this property.

**Review by nicknezis — CHANGES_REQUESTED** [review]

**Review by nicknezis — APPROVED** [review]
Looks good. For anyone viewing this PR, the bookie format init container was removed because the newer version of Bookkeeper that Heron now uses includes the initialization logic that was not previously available.

**Comment by joshfischer1108** [comment]
LGTM, Thanks for implementing @wromansky .

File: deploy/kubernetes/helm/templates/bookie.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@
 
 ## A reference of https://github.com/apache/incubator-pulsar/blob/master/kubernetes/google-container-engine/bookie.yaml
 
+
+
```
Comment (joshfischer1108) [inline deploy/kubernetes/helm/templates/bookie.yaml:23]: Clean up this white space if you will. Not a blocker though.

**Review by joshfischer1108 — APPROVED** [review]

File: deploy/kubernetes/helm/templates/bookie.yaml (hunk 2 above)
Comment (nicknezis) [inline deploy/kubernetes/helm/templates/bookie.yaml:23]:
```suggestion
```

**Pull request closed without merging**